builds/mariner/SPECS/aziot-edge/aziot-edge.spec

```diff
@@ -16,7 +16,6 @@ URL:            https://github.com/azure/iotedge
 
 %{?systemd_requires}
 BuildRequires:  systemd
-BuildRequires:  rust = 1.47.0
```

Comment:
We should still specify the Rust version here to be `1.60`. The PR (https://github.com/Azure/iotedge/commit/f9c174f98ec4433c4f5ad4716b27df8cb30dc993) has updated it to be `>1.59` for the Mariner OS. I doubt if we will have another Rust update pinning it for the upcoming 1.3 release branch.

Reply:
this would be checked into release/1.2 which i think is still on rust 1.47 

https://github.com/Azure/iotedge/blob/release/1.2/edgelet/rust-toolchain